src/coreclr/inc/utilcode.h

```diff
@@ -3447,6 +3420,9 @@ class ConfigMethodSet
     BYTE m_inited;
 };
 
+// 38 characters + 1 null terminating.
+#define GUID_STR_BUFFER_MIN_LEN (ARRAY_SIZE("{12345678-1234-1234-1234-123456789abc}"))
```

Comment:
Is `MIN_` necessary? (do we ever resize?)
`GUID_LEN` or `GUID_BUFFER_LEN` might be sufficient.

Reply:
I added it because we include the null and I didn't want people to think this is the requirement in all cases. If others think this is too much, I'd prefer to keep the `GUID_STR_BUFFER_LEN` at least because this is encoding agnostic and `GUID_BUFFER_LEN` or `GUID_LEN` is rather ambiguous.

Reply:
I'd prefer it without `MIN_`. I think `GUID_STR_BUFFER_LEN` makes sense.

I actually find `MIN` a bit confusing - especially if it is meant to indicate that the terminating null is included, as my assumption about something with `MIN` would be not including the null.